main.js

```diff
@@ -27,25 +28,28 @@ class App extends React.Component {
   state = {
     username: undefined,
   };
+
   componentDidMount() {
-    Linking.addEventListener('url', this._handleAuth0Redirect);
+    Linking.addEventListener('url', this._handleAuth0Redirect.bind(this));
   }
 
-  _loginWithAuth0 = async () => {
+  async _loginWithAuth0() {
```

Comment:
Why You've changed arrow functions into normal ones? When you are changing this from arrow you do not have a `this` context. That's why you have to `.bind(this)` at the top in `this._handleAuth0Redirect` call :)

Reply:
Good to know, I just would to uniformise all functions style ^^ So back to arrow functions!